appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmMetadataCache.java

```diff
@@ -48,6 +48,12 @@ public VmMetadataCache() {
     cache = new HashMap<String, String>();
   }
 
+  public String putMetadata(String path,String value) {
```

Comment:
space after the comma.

Also, seems like all synchronization is in the level of the map operations.
In that case we could use instead ConcurrentHashMap or just decorate via [Collections.synchronizedMap](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#synchronizedMap(java.util.Map))